fastlane-plugin-wpmreleasetoolkit.gemspec

```diff
@@ -14,8 +14,7 @@ Gem::Specification.new do |spec|
 
   spec.required_ruby_version = '>= 2.7'
 
-  spec.files         = Dir['lib/**/*'] + %w[README.md LICENSE]
-  spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
+  spec.files = Dir['lib/**/*'] + %w[README.md LICENSE]
```

Comment:
Interesting, not sure why rubocop made us remove `spec.test_files` here… is it because that gemspec attribute has now been a deprecated by rubygems or something? 🤔 
(I indeed don't see `test_files` documented in https://guides.rubygems.org/specification-reference/ 🤔 )

Reply:
@AliSoftware Yep, that's it. This is the rule that removed the line: 

https://docs.rubocop.org/rubocop/cops_gemspec.html#gemspecdeprecatedattributeassignment